docs/key_portability.md

```diff
@@ -0,0 +1,51 @@
+# Trade-offs around Key Portability and Key Import
+
+This page discusses the trade-offs between key portability and key security. Key import is a feature that supports key portability.
+
+Key portability refers to moving keys between different systems or devices. This can be useful for backup, disaster recovery, and migrating to a new system. On the other hand, key security refers to the ability to securely store and manage keys to prevent unauthorized access.
+
+There are trade-offs to consider when designing a system that must balance key portability and security. Here are a few key considerations:
+
+## Security
+
+Security is always the top priority when it comes to cryptographic keys. A system that prioritizes key portability over key security may sacrifice security to make it easier to move keys between systems. On the other hand, a system that prioritizes key security over key portability may require more secure methods for importing keys, such as secure channels or physical transfer.
+
+## Compatibility
+
+When it comes to key portability, compatibility is an important consideration. Keys should be compatible between different systems and devices and should be stored in a format that is widely recognized and accepted. Key import may require additional steps to ensure compatibility, such as converting keys to a different format or ensuring that the importing system supports the same cryptographic algorithms as the exporting system.
+
+## Complexity
+
+Key portability can add complexity to a system, as it may require additional steps and processes to ensure that keys are properly moved between systems. Key import can also add complexity, as it may require additional security measures and validation steps to ensure the imported keys are legitimate.
+
+## Use Cases
```

Comment:
it's all quite vague. I'm not sure what to suggest. What problem is this document trying to solve?